sockeye/inference.py

```diff
@@ -581,7 +592,16 @@ def translate(self, trans_input: TranslatorInput) -> TranslatorOutput:
                                     attention_matrix=np.asarray([[0]]),
                                     score=-np.inf)
 
-        return self._make_result(trans_input, *self.translate_nd(*self._get_inference_input(trans_input.tokens)))
+        if len(trans_input.tokens) > self.max_input_length:
+            logger.warning("Input (%d) exceeds max bucket size (%d). Splitting into chunks of size %d.",
```

Comment:
bucket size is actually a bit confusing to the user I think. Can we rename to max input length or something?

Reply:
yeah, makes sense.